src/fstab-generator/fstab-generator.c

```diff
@@ -551,6 +551,20 @@ static int parse_fstab(bool initrd) {
                         continue;
                 }
 
+                if (initrd && fstab_test_option(me->mnt_opts, "bind\0" "rbind\0")) {
+                        char *initrd_what;
+
+                        initrd_what = normalize_path(what);
```

Comment:
BTW, relative path is a valid fstab setting?

Reply:
It seemed to work before :), at least with `mount`.  I don't want to call it valid, but I chose to not break it.  I don't remember if I tested `systemctl start foo.mount` before this change though.

Reply:
Hmm. Then, such a path is relative to where?

Reply:
For your curiosity: the path is relative to the current directory of the mount process.  This means it works fine (e.g. during boot), up until the point a user manually uns `mount` from a directory other than `/` :).